explainaboard/builders/conditional_generation.py

```diff
@@ -239,13 +239,13 @@ def get_overall_performance(
         for metric_name in sys_info.metric_names:
 
             overall_value = self.score_dic["corpus_level"]["corpus_" + metric_name]
-            confidence_score_low = 0.0
-            confidence_score_up = 0.0
+            confidence_score_low = None
```

Comment:
Perhaps we can remove this variable and do (same for high):
```
Performance = (
...
confidence_score_low=None
...
)
```
This is more consistent with your code in other places. 😃